### ISSUE TITLE
Managed dependencies not available

### ISSUE THREAD

**Issue by ryan-efendy** [GoogleCloudPlatform/cloud-code-vscode#498]
Issue Type: <b>Bug</b>

![image](https://user-images.githubusercontent.com/12979345/141352239-876cbbac-0e52-459b-ac52-33034faf49b5.png)

*Context*
- Managed Dependencies: I tried both, initially it was `on` and the screenshot ☝🏼 is the error I got on my notification then I updated it to `off` and tried to install the cloud SDK via homebrew `brew install --cask google-cloud-sdk` and I got the error below 👇🏼
- Cloud SDK Version:
```
> gcloud --version                                                                             
ERROR: gcloud failed to load: module 'collections' has no attribute 'Mapping'
    gcloud_main = _import_gcloud_main()
    import googlecloudsdk.gcloud_main
    from googlecloudsdk.calliope import cli
    from googlecloudsdk.calliope import actions
    from googlecloudsdk.calliope import markdown
    from googlecloudsdk.calliope import usage_text
    from googlecloudsdk.calliope import parser_arguments
    from googlecloudsdk.calliope import parser_completer
    from googlecloudsdk.core.console import progress_tracker
    class _BaseStagedProgressTracker(collections.Mapping):

This usually indicates corruption in your gcloud installation or problems with your Python interpreter.

Please verify that the following is the path to a working Python 2.7 or 3.5+ executable:
    /Users/ryanefendy/.pyenv/versions/3.10.0/bin/python

If it is not, please set the CLOUDSDK_PYTHON environment variable to point to a working Python 2.7 or 3.5+ executable.

If you are still experiencing problems, please reinstall the Cloud SDK using the instructions here:
    https://cloud.google.com/sdk/
```
I'm using pyenv to manage my python versions and the path `/Users/ryanefendy/.pyenv/versions/3.10.0/bin/python` is what I set my global python as.
```
> pyenv
pyenv 2.2.0
Usage: pyenv <command> [<args>]

> which python
/Users/ryanefendy/.pyenv/shims/python

> pyenv versions
  system
* 3.10.0 (set by /Users/ryanefendy/.pyenv/version)
```

and I have specified `CLOUDSDK_PYTHON` below, but still getting the error ☝🏼
```
export CLOUDSDK_PYTHON=$(which python)
```

- Skaffold Version: v1.34.0 (brew install skaffold)
- Minikube Version: v1.24.0 (brew install minikube)

Extension version: 1.15.1
VS Code version: Code - Insiders 1.63.0-insider (c1096372119bcf83b9bd8ccee96dcf266f35de76, 2021-11-11T13:17:33.094Z)
OS version: Darwin arm64 20.6.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|15, 13, 9|
|Memory (System)|16.00GB (0.16GB free)|
|Process Argv|--crash-reporter-id f1bfd64f-cc52-4aa7-9bd5-ab82475bf95f --crash-reporter-id f1bfd64f-cc52-4aa7-9bd5-ab82475bf95f|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by ryan-efendy** [comment]
@sivakku @sujit-kamireddy can you guys take a look?

**Comment by ryan-efendy** [comment]
Solved by downgrading python from 3.10 to 3.8. Python 3.10 *collections* doesn't have *Mapping* hence I got this`ERROR: gcloud failed to load: module 'collections' has no attribute 'Mapping'`; downgraded to the recommended version (3.5 to 3.8, 3.7) https://cloud.google.com/sdk/docs/install#mac and it works.

```
> gcloud version
Google Cloud SDK 364.0.0
bq 2.0.71
core 2021.11.05
gsutil 5.4
```

**Comment by sujit-kamireddy** [comment]
Thank you @ryan-efendy
